src/tabpfn_extensions/hpo/tuned_tabpfn.py

```diff
@@ -285,15 +285,23 @@ def objective(params):
                 else:  # Regression
                     y_pred = model.predict(X_val)
                     if self.metric == MetricType.RMSE:
-                        score = -mean_squared_error(
-                            y_val, y_pred, squared=False
-                        )  # Negative RMSE
+                        score = -score_regression(
```

Comment:
I would say we can leave it as before, or pass the metric directly to score_regression.